lib/importers/upload_importer.rb

```diff
@@ -19,6 +19,18 @@ def self.update_usage_count(commons_uploads)
     end
   end
 
+  def self.find_deleted_files(commons_uploads)
+    Utils.chunk_requests(commons_uploads) do |file_batch|
+      deleted_files = Commons.find_missing_files file_batch
```

Comment:
That's what I thought. Then can we do `deleted_files.each { |file| file.update_attribute(:deleted, true) }`?